disnake/http.py

```diff
@@ -1957,7 +1957,7 @@ def edit_role(
         valid_keys = (
             "name",
             "permissions",
-            "color",
+            "colors",
```

Comment:
There are no guarantees in the library about this being a stable interface, but it wouldn't hurt to keep both here, in case anyone is calling `HTTPClient` methods directly nonetheless.

Reply:
https://github.com/DisnakeDev/disnake/pull/1312/commits/2ba39f291bf3b4f21acaeb4f6be3c2a16cab24ab